src/Ripple/TouchRipple.js

```diff
@@ -26,6 +26,9 @@ export const styleSheet = createStyleSheet('MuiTouchRipple', () => ({
 
 export default class TouchRipple extends Component {
   static propTypes = {
+    /**
+     * TODO: What is this?
```

Comment:
I thought they were always supposed to be centered at the interaction: "Ink ripples confirm user input by immediately expanding outward from the point of touch. "?https://material.io/guidelines/motion/material-motion.html#material-motion-how-does-material-move

Doesn't look like it's used anywhere other than an option in `ButtonBase` (`<Button>` doesn't explicitly define this prop), but I'll document it as-is. And we can decide later whether it's a useful feature.